### PR TITLE
Colour + compass for turning

### LINKED ISSUE #30: Compass sensor
- [x] Add a compass sensor to the bot
- [x] Read directions
- [x] Use direction to turn precisely

### LINKED ISSUE #31: Farbsensor

### LINKED ISSUE #47: Color Kalibirierung
Kalibrierung nicht über absolute Threshholds, sondern folgendermaßen: 
grün > faktor_1 * blue , wobei Faktor kalibriert wird
Analog für Rot das Gleiche

### PR DESCRIPTION
**Pull request by FlederJ**
adjusted colour delays for moving
added lights to show which way it will turn
kreuzung logic adapted to frontalLine and sideLine scenario without straighten()
color calibrates, closes #47 
close #31 
added turning by degrees with compass, close #30

### CODE REVIEW

**Review by zockerhuhn — CHANGES_REQUESTED** [review]
Change red to work

File: programs/Main/kreuzung.h
Comment (zockerhuhn) [inline programs/Main/kreuzung.h:1]: Use factor for straight() in line 26

**Review by zockerhuhn — CHANGES_REQUESTED** [review]
Either red isn't working yet or the code contains a lot of unnecessary comments

File: programs/Main/kreuzung.h
Comment (FlederJ) [reply programs/Main/kreuzung.h:1]: done

**Review by zockerhuhn — APPROVED** [review]
seems good now

File: programs/Main/Main.ino
[the remark this answers is not in the document]
Comment (FlederJ) [reply programs/Main/Main.ino:1]: please state which comments you consider to be unnecessary

[the remark this answers is not in the document]
Comment (FlederJ) [reply programs/Main/Main.ino:1]: but the important thing is that everything seems to work for now (except the bumpers), and there is a possibility for a softlock, but it's mostly correct

**Pull request merged**